### PR TITLE
[csstools] Convert slides to reveal.js 

### PR DESCRIPTION
**Pull request by maxjchung**
# Summary

Fixes issue #236. Changes include: 

* Added missing `</***>` tags as needed
* Normalized indentation to spaces
* Added line breaks to keep some code examples in viewport.
* Deleted commented-out material (see 0942731 and 2907f76)

cc @gdisf/admins

### CODE REVIEW

File: csstools/index.html [hunk 1]
```diff
@@ -1,786 +1,392 @@
...
+    <!-- light editor -->
+    <link rel="stylesheet" href="../common/lib/css/light.css" />
+    <!-- dark editor<link rel="stylesheet" href="lib/css/dark.css"> -->
+    <!-- If use the PDF print sheet so students can print slides-->
+
+    <link rel="stylesheet" href="../common/css/print/pdf.css" type="text/css" media="print" />
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
     <!--[if lt IE 9]>
-      <meta http-equiv="X-UA-Compatible" content="IE=edge;chrome=1" />
+    <script src="lib/js/html5shiv.js"></script>
```
Comment (brendajin) [inline csstools/index.html]: can you link to the file in `common/` here?

Comment (maxjchung) [inline csstools/index.html]: Fixed in c459f02534fa00a5fb76ef197927212182f1ad2f.

File: csstools/index.html [hunk 2]
```diff
@@ -1,786 +1,392 @@
...
+            <li>Responsiveness and cross browser consistency for free.</li>
+            <li>Modular and reusable components.</li>
+            <li>Usually include a reset, a grids system, styles for typography and forms. Some include support for mobile/print.</li>
+          </ul>
+        </section>
+
+        <section>
+          <h2>CSS Frameworks</h2>
+          <p>With <a href="http://v4-alpha.getbootstrap.com/">Twitter Bootstrap</a>:</p>
+          <pre>
```
Comment (brendajin) [inline csstools/index.html]: Can you make these blocks use `<pre><code></code></pre>` for formatting? that will add a background color.

The other thing to note is that the formatting will take all the literal whitespace, so you'll want to un-indent these sections.

File: csstools/index.html [hunk 3]
```diff
@@ -1,786 +1,392 @@
...
+            https://developer.mozilla.org/en/CSS/@media#Media_groups
+            https://developer.mozilla.org/en/CSS/Media_queries
+          </p>
+        </section>
+
+        <section>
+          <h2>Print Styles</h2>
+          <p>All modern browsers support the "print" media type for specifying print styles.</p>
+          <pre>
+            @media print {
```
Comment (brendajin) [inline csstools/index.html]: here's an example of how this segment can go:
```
<pre><code>@media print {
  body {
    background: white;
  }
  #header, #footer {
    display: none;
  }
  .section {
    page-break-inside: avoid;
    page-break-after: always;
  }
}
</code> </pre>
```

## Before
screen shot 2016-07-30 at 1 18 58 pm

## After
screen shot 2016-07-30 at 1 20 18 pm

File: csstools/index.html (hunk 2 above)
Comment (maxjchung) [inline csstools/index.html]: Yup! Done in pre07bf23a.

I kept the `<pre><code>` and `</code></pre>` tags indented at the same level, un-indenting the code within.

File: csstools/index.html [hunk 4]
```diff
@@ -1,786 +1,389 @@
...
+
+
+        <section>
+          <h2>Resolution Queries</h2>
+          <p>Modern mobile browsers support media queries which depend on the <code>device-width</code> and/or
+          <code>orientation</code>:</p>
+          <p>Targeting iPads:</p>
+          <pre><code>
+@media only screen and (min-device-width : 768px)
+                   and (max-device-width : 1024px) {
```
Comment (brendajin) [inline csstools/index.html]: the lines after the first one should move over a bit to the left here, and the last `}` should be flush with the left

File: csstools/index.html [hunk 5]
```diff
@@ -1,786 +1,389 @@
...
-    </div>
-
-
+       only screen and (-o-min-device-pixel-ratio: 3/2),
+       only screen and (min--moz-device-pixel-ratio: 1.5),
+       only screen and (min-device-pixel-ratio: 1.5) {
+         #logo {
+           background-image:url(images/icon@2x.png);
+         }
+       }
```
Comment (brendajin) [inline csstools/index.html]: same thing here. the  `#logo` should be farther in and the last `}` should be flush left. see negative diff for reference

**Comment by brendajin** [comment]
two minor whitespace change requests, then this looks ready to merge!

**Comment by maxjchung** [comment]
Does 2149800383f1de2797e74a6c1b7839bf57d2fe00 get it right?

**Comment by brendajin** [comment]
Looks great, thank you!

**Pull request merged**